taca/server_status/server_status.py

```diff
@@ -185,10 +186,15 @@ def get_uppmax_cpu_hours():
 
         # split line into a list
         project = proj.split()
-        # creating objects
-        project_dict["project"] = project[0]
-        project_dict["cpu hours"] = project[1]
-        project_dict["cpu limit"] = project[2]
+        # sometimes it returns empty strings or something strange
+        try:
+            # creating objects
+            project_dict["project"] = project[0]
+            project_dict["cpu hours"] = project[1]
+            project_dict["cpu limit"] = project[2]
+            result[project[0]] = project_dict
+        except Exception, e:
+            logging.error(e.message)
+            continue
 
```

Comment:
I don't think so. It's not a dictionary, it's a list.
With a random length.  
An this list is created from a string which comes as an output of `/sw/uppmax/bin/projinfo` command. Which can return just anything. Which can cause an exception.